hw/net/vhost_net.c

```diff
@@ -436,7 +436,9 @@ VHostNetState *get_vhost_net(NetClientState *nc)
 
     switch (nc->info->type) {
     case NET_CLIENT_DRIVER_TAP:
-        vhost_net = tap_get_vhost_net(nc);
+        if (!nc->vhost_net_disabled) {
```

Comment:
So we had:

1) vhost_net_start()/vhost_net_stop()
2) A dedicated vhost_started variable in VirtIONet 

Any reason for introducing another one at NetClientState structure?

Reply:
My explanation is below.